### DISCUSSION TITLE
PostgreSQL Debug not working in DBeaver 7.2.3 and PostgreSQL 12.3

### DISCUSSION THREAD

**Discussion by danieladianu** [dbeaver/dbeaver#11258] [Q&A]
#### System information: 
- Operating system (distribution) and version: Windows 10
- DBeaver version: 7.2.3
- Additional extensions: DBeaver Debug
#### Connection specification:
- Database name and version: PostgreSQL 12.3
- Driver name: 
![image](https://user-images.githubusercontent.com/73593754/97439141-76f6ae00-192e-11eb-9a98-b284a9a5a947.png)

- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
I cannot use the debugger in DBeaver 7.2.3 installed in Windows 10 and PostgreSQL 12.3 server installed on Centos 7.
When I try to debug a simple function I get the error:

"PostgreSQL debug plugin is not installed on the server.
Refer to this WIKI article for installation instructions:
https://github.com/dbeaver/dbeaver/wiki/PGDebugger#installation"

![image](https://user-images.githubusercontent.com/73593754/97440357-0781be00-1930-11eb-8763-ac26f4d482dc.png)

The  plugin_debugger (pldbgapi) extension was installed on the server-side and it can be used while calling the debug functions in other tools like OmniDB and Navicat. So I can assume that everything on the server-side is configured properly.

![image](https://user-images.githubusercontent.com/73593754/97439379-c3da8480-192e-11eb-945a-8a9c78951993.png)

The debug plugin was also installed in DBeaver.

![image](https://user-images.githubusercontent.com/73593754/97439407-cccb5600-192e-11eb-8b41-59ff7350c60e.png)

#### Steps to reproduce, if exist:
Install pldbgapi extension on PostgreSQL database server
Install debug plugin in DBeaver
Try to debug a function

**Comment by serge-rider** [comment]
Please refer to https://github.com/dbeaver/dbeaver/wiki/PGDebugger
It describes how to install debugger on server-side
